CodeMaid/Helpers/CodeItemTypeComparer.cs

```diff
@@ -139,6 +168,23 @@ private static int CalculateReadOnlyOffset(BaseCodeItem codeItem)
             return codeItemField.IsReadOnly ? 0 : 1;
         }
 
+        private static string NormalizeName(BaseCodeItem codeItem)
+        {
+            string name = codeItem.Name;
+            var interfaceItem = codeItem as IInterfaceItem;
+            if ((interfaceItem != null) && interfaceItem.IsExplicitInterfaceImplementation)
+            {
+                // Try to find where the interface ends and the method starts
+                int dot = name.LastIndexOf('.') + 1;
+                if ((dot > 0) && (dot < name.Length))
```

Comment:
I do normally as well - this one slipped by as it evolved into that (initially it was just a check for -1, but then I added to the index and then I thought I best be safe about index out of bounds)